src/naming.jl

```diff
@@ -349,7 +349,8 @@ end
     ntuple2dict(nt) -> dict
```

Comment:
add in the docstring the `[DT = Dict,] nt` to make this visible for others?

Reply:
I used `[type = Dict,]` to keep consistent with `struct2dict` docstring.